setup-etcd/action.yml

```
@@ -0,0 +1,59 @@
+name: Setup etcd
+description: Set up an etcd store from a release archive
+
+inputs:
+  version:
+    description: Release archive version (for example, v3.5.10)
+    required: false
+    default: v3.5.12
+  platform:
+    description: Release archive platform (amd64 or arm64)
+    required: false
+    default: amd64
+  install-prefix:
+    description: Release archive install directory
+    required: false
+    default: ${{ github.workspace }}/.etcd/bin/
+
+runs:
+  using: composite
+  steps:
+    - run: |
```

What do you think about `name`?
```suggestion
- name: Download ...
  run: |
```
I think it will look clearer.

---

Fixed.